packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -131,7 +134,14 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
     });
   }
 
-  if (isNotACompanyEmail(orgOwnerEmail)) {
+  if (isPlatform && !verifiedUser) {
```

Comment:
Since we now allow everyone to create a platform team chances of us getting spammed have increased, hence create an additional check just for platform to make sure the user who's trying to create a platform team has his email verified and onboarding done